bootstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# make bash behave
+set -euo pipefail
+IFS=$'\n\t'
+
+# call PostgreSQL's ENTRYPOINT script
+./docker-entrypoint.sh postgres
+
+# create worker list file link and Citus extension
+./000-symlink-workerlist.sh
```

Comment:
Hm… I think we should split the `symlink-workerlist` part into two scripts: the parts that need to run as root, and the parts that can run after the PostgreSQL image takes over.